spec/classes/init_spec.rb

```diff
@@ -53,6 +56,9 @@
           }}
           it { is_expected.to contain_class('auditd::config::logging') }
           it { is_expected.to contain_class('auditd::config::logging').that_notifies('Class[auditd::service]') }
+          it { is_expected.to contain_class('auditd::config::audisp::syslog') }
+          it { is_expected.to contain_class('rsyslog') }
```

Comment:
So, the `rsyslog` class is optional, but don't we want the `rsyslog` class to be in `metadata.json` if this module has the potential to use it?

If not, we need to update the docs to let people know to install the `rsyslog` module prior to trying to use that feature, or have a built in warning or something.

Reply:
Would an error saying that the `rsyslog` class can't be found be sufficient?

Reply:
I would add it to the README.md, since it's only if rsyslog is enabled and only relevant for the puppet module install case